test/converter.js

```diff
@@ -112,13 +112,16 @@ async function testAttrbuteType (test, jsonType, graphQLType, options) {
     type: 'object',
     properties: {
       attribute: { type: jsonType }
-    }
+    },
+    description: 'Description'
   };
 
   const expectedType = `
+  "Description"
```

Comment:
Is this a valid syntax for GraphQL comments and documentation?

Reply:
I think yes, refer this https://github.com/graphql/graphql-spec/issues/420

Reply:
Wow, I'm surprised I didn't know about that. The full spec is here: https://graphql.github.io/graphql-spec/draft/#sec-Descriptions